src/ftrace.c

```diff
@@ -48,8 +49,15 @@ static int resolve_hook_address(struct ftrace_hook *hook)
   return 0;
 }
 
-static void notrace fh_trace_thunk(unsigned long ip, unsigned long parent_ip, struct ftrace_ops *ops, struct pt_regs *regs)
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))
+static void notrace fh_trace_thunk(unsigned long ip, unsigned long parent_ip, struct ftrace_ops *ops,struct pt_regs *regs)
```

Comment:
Can you add the missing space before the last argument?

Reply:
fixed.